spec/safe_statements_spec.rb

```diff
@@ -1279,6 +1279,39 @@ def up
           end
         end
 
+        describe "raw_remove_check_constraint" do
+          before(:each) do
+            setup_migration = Class.new(migration_klass) do
+              define_method(:up) do
```

Comment:
Nitpick: I think this could just be `def up`

Reply:
if we change it to be `def up` we don't have access to the `constraint_name` local variable anymore.

Reply:
Let's add a comment explaining that.

Reply:
Ah that makes sense. And a comment makes sense too.

I've abused class attributes to get around these scoping problems when doing dynamic class stuff: https://github.com/braintree/pg_ha_migrations/blob/77c3addf776173f113a28139df5e3141a49b74db/spec/safe_statements_spec.rb#L44-L78

But what you've done is better

Reply:
Added a comment!